x-pack/plugins/security_solution/server/search_strategy/security_solution/factory/response_actions/actions/query.all_actions.dsl.ts

```diff
@@ -12,16 +12,28 @@ import { OSQUERY_ACTIONS_INDEX } from '@kbn/osquery-plugin/common/constants';
 import type { ActionRequestOptions } from '../../../../../../common/search_strategy/security_solution/response_actions';
 import { ENDPOINT_ACTIONS_INDEX } from '../../../../../../common/endpoint/constants';
 
+const EndpointFieldsLimited = [
+  'EndpointActions.action_id',
+  'EndpointActions.input_type',
+  'EndpointActions.expiration',
+  'EndpointActions.data.command',
+];
+
 export const buildResponseActionsQuery = ({
   alertIds,
   sort,
+  canAccessEndpointActionsLogManagement,
 }: ActionRequestOptions): ISearchRequestParams => {
+  const fields = !canAccessEndpointActionsLogManagement
+    ? ['@timestamp', 'action_id', 'input_type', ...EndpointFieldsLimited]
+    : [{ field: '*' }, { field: 'EndpointActions.*', include_unmapped: true }];
```

Comment:
nit, small readability thing: I think ternaries and if statements are easier to read if we don't negate the condition when it's unnecessary. `"canAccess... -> then this -> otherwise that"` instead of `"NOT canAccess -> ..."`. what do you think?

Reply:
Hey, good point regarding accesing authz from search strategy, I'll try to avoid passing that from Frontend client, thanks!
Regarding the ternary: Ok this makes sense ;) Thanks!